api/Element.json

```diff
@@ -1167,10 +1167,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Element/attributes",
           "support": {
             "chrome": {
-              "version_added": "26"
+              "version_added": "1"
```

Comment:
For anything on `Element` where the supported version is being lowered, can you confirm it was always on `Element` and not just `HTMLElement`? I would expect a few cases of moves between `HTMLElement` and `Element`, some of which will require `partial_implementation` and notes like the ones added in https://github.com/mdn/browser-compat-data/pull/6865

Reply:
Good call -- it looks like there were only two adjusted features that were initially implemented on the `HTMLElement` API.  https://github.com/foolip/mdn-bcd-collector/pull/699 will fix this so we won't get conflicting results.